docs/product-assembly_business-optimizer/src/main/asciidoc/main.adoc

```diff
@@ -15,43 +14,55 @@ As a business rules developer, you can use the {PLANNER} to find the optimal sol
 Use this document to set up and configure {PLANNER} functionality.
 
 .Prerequisite
-The *Employee Rostering* sample project has been created in {CENTRAL} to demonstrate Business Optimizer capabilities. To view this project, you must have {PRODUCT} installed. For more information about setting up and installing {PRODUCT} and {CENTRAL}, see _Getting started with decision services_.
-//@doc-link:Add link to Getting Started assembly above. 
+The *Employee Rostering* sample project has been created in {CENTRAL} to demonstrate Business Optimizer capabilities. To view this project, you must have {PRODUCT} installed. For more information about setting up and installing {PRODUCT} and {CENTRAL}, see {URL_GETTING_STARTED_DECISION_SERVICE}[{GETTING_STARTED_DECISION_SERVICE}].
 
 
 // Modules
-= Solving planning problems using Business Optimizer
+= Solve planning problems using Business Optimizer
 include::product-business-resource-planner-guide/optimizer-about-business-optimizer-con.adoc[leveloffset=+1]
 include::product-business-resource-planner-guide/optimizer-NP-complete-vs-NP-hard-con.adoc[leveloffset=+2]
 include::product-business-resource-planner-guide/optimizer-planning-problem-con.adoc[leveloffset=+1]
 include::product-business-resource-planner-guide/optimizer-search-space-con.adoc[leveloffset=+2]
 include::product-business-resource-planner-guide/optimizer-hard-vs-soft-constraints-con.adoc[leveloffset=+2]
 
-
-= Downloading and installing Business Optimizer
+= Download and install Business Optimizer
 include::product-business-resource-planner-guide/optimizer-download-proc.adoc[leveloffset=+1]
 include::product-business-resource-planner-guide/optimizer-engine-installation-proc.adoc[leveloffset=+1]
 
-= Configuring the {KIE_SERVER}
+= Running the examples
```

Comment:
Should this be "Run the examples" for consistency?

Reply:
I'm taking this back. Looking at the whole PDF I can see that the heading forms are mixed evenly (solve, download, configure, vs. running, deploying, managing, etc.) so maybe consistency is not a goal here. (If it is, then please update all headings to be in the same form.)

Reply:
+1 on consistency.

Reply:
Yep, fixed.